### DISCUSSION TITLE
Missing tabs in table properties screen

### DISCUSSION THREAD

**Discussion by CoreshLtd** [dbeaver/dbeaver#10613] [Q&A] [answered]
After upgrading DBeaver to 7.3.0 version, when I do "Open Declaration" on a table (i.e. by the F4 key), there are now only 3 tabs at the left: 

1. Columns
2. Statistics
3. DDL

How can I fix that?
![1](https://user-images.githubusercontent.com/75490232/101181879-1443b100-3656-11eb-836e-6b8f3afc7f6b.png)

**Comment by kseniiaguzeeva** [comment, marked as answer]
Most probably Simply view is selected.
Please open the connection context menu and select Connection view>Advanced

**Comment by CoreshLtd** [comment]
@kseniiaguzeeva you are absolutely correct!
Issue was because "Connection view ->Simple" was  selected, probably by mistake.
Issue is now fixed after changing to Connection view-->Advanced.
Thank you :)

**Comment by kseniiaguzeeva** [comment]
For pleasure)
